### PR TITLE
Fixed String Formatting issue closes #2343, #2345, #2347

### LINKED ISSUE #2343: Fix string formatting issue for Test : tests.foreman.ui.test_gpgkey.GPGKey.test_consume_content_1
Failing due to:
Unicode formatting issue for string in AssertEqual function of Test.

StackTrace:
```
File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/shiningpanda/jobs/abb3bf3f/virtualenvs/d41d8cd9/lib/python2.7/site-packages/ddt.py", line 146, in wrapper
    return func(self, *args, **kwargs)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-ui/tests/foreman/ui/test_gpgkey.py", line 456, in test_consume_content_1
    .format(result.stderr, result.return_code)
'\'ascii\' codec can\'t encode character u\'\\u201c\' in position 298: ordinal not in range(128)
```

### LINKED ISSUE #2345: Fix string formatting issue for Test : tests.foreman.ui.test_activationkey.ActivationKey.test_multiple_activation_keys_to_system
Failing due to:
Unicode formatting issue for string in AssertEqual function of Test.

StackTrace:
```
File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-ui/tests/foreman/ui/test_activationkey.py", line 1388, in test_multiple_activation_keys_to_system
    .format(result.stderr, result.return_code)
'\'ascii\' codec can\'t encode character u\'\\u201c\' in position 298: ordinal not in range(128)
```

### PR DESCRIPTION
**Pull request by jyejare**
Fixed String formatting issue.

Logs:
```
[root@jyejare robottelo]# nosetests -m test_multiple_activation_keys_to_system ./tests/foreman/ui/test_activationkey.py
.
----------------------------------------------------------------------
Ran 1 test in 539.167s

OK
```

### CODE REVIEW

File: tests/foreman/ui/test_activationkey.py [hunk 1]
```diff
@@ -1384,7 +1384,7 @@ def test_multiple_activation_keys_to_system(self):
             )
             self.assertEqual(
                 result.return_code, 0,
-                "failed to fetch katello-ca rpm: {0}, return code: {1}"
+                u"failed to fetch katello-ca rpm: {0}, return code: {1}"
```
Comment (elyezer) [inline tests/foreman/ui/test_activationkey.py]: We are safe to drop the assert message, we already have these information on robottelo log.

**Comment by elyezer** [comment]
ACK, pending comment.

**Comment by jyejare** [comment]
Test Reults:
```
[root@jyejare robottelo]# nosetests -m test_consume_content_1 ./tests/foreman/ui/test_gpgkey.py
.
----------------------------------------------------------------------
Ran 1 test in 299.731s

OK
[root@jyejare robottelo]# nosetests -m test_multiple_activation_keys_to_system ./tests/foreman/ui/test_activationkey.py
.
----------------------------------------------------------------------
Ran 1 test in 532.880s

OK
[root@jyejare robottelo]# nosetests -m test_usage_limit ./tests/foreman/ui/test_activationkey.py
.
----------------------------------------------------------------------
Ran 1 test in 311.397s

OK
[root@jyejare robottelo]# nosetests -m test_positive_delete_activation_key_5 ./tests/foreman/ui/test_activationkey.py
.
----------------------------------------------------------------------
Ran 1 test in 361.491s

OK
```

**Comment by oshtaier** [comment]
APT

**Comment by elyezer** [comment]
ACK

**Comment by elyezer** [comment]
Karma given.

**Pull request merged**